src/elements/content-uploader/ItemAction.js.flow

```diff
@@ -23,34 +14,60 @@ import {
     STATUS_COMPLETE,
     STATUS_ERROR,
 } from '../../constants';
+
+import messages from '../common/messages';
+
 import type { UploadStatus } from '../../common/types/upload';
 
 import './ItemAction.scss';
+import { Size5, SurfaceStatusSurfaceSuccess } from '@box/blueprint-web-assets/tokens/tokens';
 
-const ICON_CHECK_COLOR = '#26C281';
-
-type Props = {
-    error?: Object,
+type ItemActionProps = {
+    error?: any,
     intl: IntlShape,
     isFolder?: boolean,
     isResumableUploadsEnabled: boolean,
-    onClick: Function,
-    onUpgradeCTAClick?: Function,
+    onClick: any,
+    onUpgradeCTAClick?: any,
     status: UploadStatus,
 };
 
+const getIconWithTooltip = (
+    icon: React.ReactNode,
+    isDisabled: boolean,
+    isLoading: boolean,
+    onClick: any,
```

Comment:
can we type this

Reply:
This was purposely converted to an any so that it can be converted into TS, all type changes in the `.js.flow` file was done by the codemod